crates/ibc/src/core/ics04_channel/msgs/acknowledgement.rs

```diff
@@ -20,6 +20,11 @@ impl Acknowledgement {
     pub fn is_empty(&self) -> bool {
         self.0.is_empty()
     }
+
+    // Returns the data as a slice of bytes.
+    pub fn as_bytes(&self) -> &[u8] {
```

Comment:
Nit: we already have a `AsRef<[u8]>` impl below so can we use that instead of defining this new public method?

Reply:
I find `.as_ref()` not descriptive enough, and following the std practice also provided an intrinsic method with a clearer name.
In general, `AsRef` makes the type more usable as a parameter in generic functions that use that bound, while the intrinsic `.as_slice()`/`.as_bytes()` method is more readable in code using the concrete type.